silkrpc/core/evm_executor.hpp

```diff
@@ -65,7 +65,7 @@ class EVMExecutor {
     const core::rawdb::DatabaseReader& db_reader_;
     const silkworm::ChainConfig& config_;
     asio::thread_pool& workers_;
-    state::RemoteBuffer buffer_;
+    state::RemoteState buffer_;
```

Comment:
After renaming the variable type `RemoteBuffer` as `RemoteState`, also the variable name `buffer_` should be renamed `remote_state_`